wordpress-mobile/AztecEditor-Android
# Advanced styling of headings

### Fix
In this PR I'm introducing advanced options for styling some elements. For example headings now can be sized and changed each one differently. Headings now support size and color. 

### Test
1. Modify the AztecText style in MainActivity to use `headingNFontSize` or `headingNFontColor` where `N` stands for heading types between `one` and `six`
2. Run the app
3. Notice the heading styling

### Review
@[USER_NAME]

Make sure strings will be translated:
